src/test/java/teammates/test/cases/common/TimeHelperTest.java

```diff
@@ -92,9 +92,9 @@ public void testEndOfYearDates() {
         cal.set(2015, 11, 30, 12, 0, 0);
         Date date = cal.getTime();
         assertEquals("30/12/2015", TimeHelper.formatDate(date));
-        assertEquals("Wed, 30 Dec 2015, 12:00 PM", TimeHelper.formatTime12H(date));
+        assertEquals("Wed, 30 Dec 2015, 12:00 NOON", TimeHelper.formatTime12H(date));
         assertEquals("Wed, 30 Dec 2015, 12:00 PM UTC", TimeHelper.formatDateTimeForComments(date));
-        assertEquals("30 Dec 12:00 PM", TimeHelper.formatDateTimeForInstructorHomePage(date));
+        assertEquals("30 Dec 12:00 NOON", TimeHelper.formatDateTimeForInstructorHomePage(date));
```

Comment:
The default display of 12 midnight will be 00: 00 AM. I think it is ok to keep 00:00 am. Also, if the time is 30 Jan, 12 MIDNIGHT, will it be confused that it should be 30 Jan, 00: 00am or 31 Jan, 00:00am?